app/controllers/spree/admin/base_controller.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/ClassLength
```

Comment:
I thought these exceptions/comments were for things we accept we will keep forever.
I think we should try at some point to make this class shorter, so I'd say this should be in rubocop manual.

Reply:
I've refactored the BaseController a bit to make it shorter.

Reply:
nice refactor👌